src/command/client/search/interactive.rs

```diff
@@ -31,6 +31,9 @@ use super::{
 };
 use crate::VERSION;
 
+const RETURN_ORIGINAL: usize = usize::MAX;
+const RETURN_QUERY: usize = usize::MAX - 1;
```

Comment:
I was about to complain about magic numbers... Except I was the one who added them in the first place 😂

Reply:
😂